run-bungeecord.sh

```diff
@@ -94,16 +94,13 @@ fi
 # Download rcon plugin
 if [ "${ENABLE_RCON^^}" = "TRUE" ] && [[ ! -e $BUNGEE_HOME/plugins/${RCON_JAR_URL##*/} ]]; then
   echo "Downloading rcon plugin"
-  if ! curl -sSL -o /tmp/${RCON_JAR_URL##*/} $RCON_JAR_URL; then
+  mkdir -p $BUNGEE_HOME/plugins/bungee-rcon
+
+  if ! curl -sSL -o "$BUNGEE_HOME/plugins/${RCON_JAR_URL##*/}" $RCON_JAR_URL; then
```

Comment:
Should this be?
```suggestion
  if ! curl -sSL -o "$BUNGEE_HOME/plugins/bungee-rcon/${RCON_JAR_URL##*/}" $RCON_JAR_URL; then
```

Reply:
no because `/plugins/bungee-rcon` its folder for `config.yml`, all plugin need go to `/plugins`

Reply:
Oh yeah, that makes sense